tests/integration/tests.py

```diff
@@ -227,13 +227,6 @@ def test_correct_data_with_get(self):
         instance = Event.objects.get()
         assert instance.message == 'hello'
 
-    @override_settings(SENTRY_ALLOW_ORIGIN='sentry.io')
-    def test_get_without_referer(self):
-        self.project.update_option('sentry:origins', '')
-        kwargs = {'message': 'hello'}
-        resp = self._getWithReferer(kwargs, referer=None, protocol='4')
-        assert resp.status_code == 403, (resp.status_code, resp.get('X-Sentry-Error'))
```

Comment:
To be fair, I'd argue that these tests should be updated to test the updated behavior and prove that they now work.